### PR TITLE
Change apt to apt-get.

### PR DESCRIPTION
**Pull request by Elliria**
This satisfies changes in **develop**, which is one of the three branches described as needing it in issue #772.

### CODE REVIEW

**Comment by Elliria** [comment]
Please merge this one, @josephj11. It's one of three pull requests that need to happen in order to complete issue #772. I thought I'd start with this change in the **develop** branch and then do each of the **beta** and **master changes separately.

**Comment by josephj11** [comment]
IDK if it's related to this PR or not, but we got [heckled](https://github.com/autokey/autokey/actions/runs/4327567057/jobs/7556347694) by Jeckyl again.

**Comment by Elliria** [comment]
I doubt it's connected to that commit. It's that documentation problem we haven't sorted out yet. When I had asked @BlueDrink9 if we should create that folder, he said it works the other way around and that it's created for us automatically when the whole thing works properly. Now it's a matter of somehow finding out how to get it to work properly, but I have no idea yet. I suppose "jeckyl" will be a good starting point for our searches.

**Comment by Elliria** [comment]
I see what caused Jekyll to pop up, but I don't know why it happened. When I went poking around in the branches on my fork today, I was prompted to sync, so I did. When I did that, I got the **patch-1** branch, which was odd.

In looking at that branch, the only thing it contains is the change I made to the **bug.yaml** file, but I made that change on the **master** branch. I have no idea why it created a separate branch, but I suspect it acted like a rebase instead of a merge. Also, that happened right before this pull request, which explains why Jekyll was triggered by this pull request.

Either way, I seem to have the power to delete the **patch-1** branch (you can see the little trash can in my screenshot), but I'm not sure if that would trigger a PR and get rid of it at your end as well. Would you like me to try getting rid of it from here or would you prefer to do it?

PowerToDelete

**Comment by josephj11** [comment]
TL;DR: I have no idea what to do here, so I'd like to know more before doing anything.

IDK what's going on here. I'm a bit hesitant to delete it because IDK if that will break something else. The stakes are a bit lower since this is the devlop branch and not main.

If we find someone to ask for help with this, they might want to see it. The point is to eitiher prevent it from happening in the future or know why it did and why that's good for some reason.

I would be more comfortable with this if we had some idea why it was created. I assume the changes are already in develop, so deleting the patch shoudn't lose anything.

I don't think you can currently do anything elevated in develop or main, so what is this patch animal if you have permission to delete it?

**Comment by josephj11** [comment]
SWAG: Maybe GitHub notiiced that develop had changed relatiive to your private branch and gave you a "patich" so you coud incorporate those changes in your branch if you wanted to.

I'm not convinced that this makes sense, but if there's anything to it, there woud aa lot on the web about it.

[This](https://www.geeksforgeeks.org/patch-operation-in-git/) shows how to intentiionally create and use a patch as above, but doesn't address something creating a patch for you.

**Comment by Elliria** [comment]
It's solved:

It turns out that the **patch-1** branch was created when I attempted to edit a file in a repository that I don't have push rights to (the AutoKey repository) instead of editing that file in my fork.

When you do that, GitHub automatically offers to create a branch with the default name of **patch-1** (or **patch-2**, etc., depending on whether you've made other patch branches before or not). You have the power to edit the branch name at that point, but it seems that I didn't. 

That branch gets created inside the repository you don't have push rights to (in this case, the AutoKey repository) if the pull request is accepted, and it's also the branch that's used as the source for the pull request.

I then got a copy of that branch the next time I synced my fork with the AutoKey repository, which was what spurred this. When I view the branches in my fork, it says I own the **patch-1** branch. Since that's the case, I can go ahead and delete it.

What I don't know is whether deleting it at my end will generate a pull request that removes it from the AutoKey repository, too, or if you'll need to manually delete it from there as well. We're about to find out, because I'm getting rid of it in my fork.

**Comment by Elliria** [comment]
I deleted it from my fork and that got rid of it from AutoKey. As Colonel John "Hannibal" Smith (George Peppard) used to say on "A-Team", "I love it when a plan comes together."

**Comment by josephj11** [comment]
Great! Another note to add to the delveloper notes: If you get this, this is what you did wrong...

**Comment by Elliria** [comment]
Done.

**Comment by josephj11** [comment]
I saw an A Team episode again a year or two ago. There were battles, but absolutely no blood and guts. It struck me how much things have changed.
